MediaBrowser.Common/Plugins/PluginManifest.cs

```diff
@@ -0,0 +1,85 @@
+#nullable enable
+using System;
+using MediaBrowser.Model.Plugins;
+
+namespace MediaBrowser.Common.Plugins
+{
+    /// <summary>
+    /// Defines a Plugin manifest file.
+    /// </summary>
+    public class PluginManifest
+    {
+        /// <summary>
+        /// Gets or sets the category of the plugin.
+        /// </summary>
+        public string Category { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the changelog information.
+        /// </summary>
+        public string Changelog { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the description of the plugin.
+        /// </summary>
+        public string Description { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the Global Unique Identifier for the plugin.
+        /// </summary>
+#pragma warning disable CA1720 // Identifier contains type name
+        public Guid Guid { get; set; }
+#pragma warning restore CA1720 // Identifier contains type name
+
+        /// <summary>
+        /// Gets or sets the Name of the plugin.
+        /// </summary>
+        public string Name { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets an overview of the plugin.
+        /// </summary>
+        public string Overview { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the owner of the plugin.
+        /// </summary>
+        public string Owner { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the compatibility version for the plugin.
+        /// </summary>
+        public string TargetAbi { get; set; } = string.Empty;
+
+        /// <summary>
+        /// Gets or sets the upper compatibility version for the plugin.
+        /// </summary>
+        public string MaxAbi { get; set; } = string.Empty;
```

Comment:
The target ABI is supposed to be the upper and lower bound on plugins, so we probably don't want yet another field that accomplishes the same thing.

Reply:
Could you please clarify as hat would mean that the plugin will only work on one specific version? 
targetAbi contains one value - meaning that if we release 10.7.2 none of the plugins will work, as they will have a targetAbi of 10.7.1

Reply:
If 10.7.1 and 10.7.2 have breaking changes- we did something majorly wrong.
The plugin targets 10.7(.0), which includes all of 10.7.*

Reply:
Just read up version comparison - and see what you mean.
Removing change.

Reply:
Did you mean to remove this property?
It's no longer used anywhere

Reply:
I swear i did a full code search and removed all instances.

Think i need a holiday!